gef-ui/server/controllers/companies-house/index.js

```diff
@@ -74,6 +49,9 @@ const validateCompaniesHouse = async (req, res) => {
      * added smeType, probabilityOfDefault, isFinanceIncreasing as blank strings
      * enables to be added to database so shows on application-details page if not exited without saving
      */
+
+    companiesHouseDetails.selectedIndustry = companiesHouseDetails.industries?.length === 1 ? companiesHouseDetails.industries[0] : null;
```

Comment:
Should we nullish coalesccing operator please to further simplify this statement, however feel free to disagree.

```javascript
companiesHouseDetails.selectedIndustry = companiesHouseDetails.industries[0] ?? null;
```

Reply:
This changes the logic - we want to set `.selectedIndustry` to `companiesHouseDetails.industries[0]` if and only if the array has length 1, whereas your suggestion would just require length >=1

Reply:
Ah yes, apologies missed it. Please ignore this comment.